Userland/Applications/Assistant/Providers.cpp

```diff
@@ -34,7 +34,8 @@ void AppResult::activate() const
         exit(1);
     }
 
-    m_app_file->spawn();
+    auto arguments_list = m_arguments.split(' ');
+    m_app_file->spawn(arguments_list.span());
```

Comment:
...continuing from previous comment: We can call `split_view()` instead of `split()`, which also avoids allocating a bunch of temporary strings.